.github/pull_request_template.md

```diff
@@ -4,3 +4,16 @@
 
 - [ ] If I changed code, I ran `yarn build` and committed resulting changes.
 - [ ] I added an example exercising this PRs functionality to `.github/workflows/test.yml` or explained why it doesn't make sense to do so.
+
+> [!IMPORTANT]
+>
+> After merging, make sure to create a new GitHub release and associated tag for this release.
+> Additionally, if this is not a breaking change, make sure to update the `v1` tag:
+>
+> ```shell
```

Comment:
Is the tag in `$TAG` manually created too? If there is, let's add the instruction to create that tag here too. I'm guessing it's something like this:

```
git checkout master
export TAG=v1.2.3
git tag -a $TAG -m "Releases $TAG"
git push origin $TAG
```

If not, then ignore me :)

Reply:
Updated the directive- I just create the releases with the GitHub release UI since that also creates a tag.